descriptions/rep/en_us.lua

```diff
@@ -474,222 +471,222 @@ EID.descriptions[languageCode].birthright ={
+	[386] = "Middle ring wisp#5% chance for tears to reroll rocks, urns, skulls or mushrooms", -- D12
 	[396] = "Cannot shoot tears#Each portal has 1 wisp", -- Ventricle Razor
-	[406] = "Random damage and fire rate", -- D8
-	[419] = "20% chance for tears to teleport enemies#Wisps flicker every 5 seconds", -- Teleport 2.0
-	[421] = "Charm fart when destroyed", -- Kidney Bean
-	[422] = "30% chance to shoot petrifying tears", -- Glowing Hour Glass
-	[427] = "Bounces harmlessly around the room, exploding when shot by Isaac#Cannot shoot tears or deal contact damage", -- Mine Crafter
-	[434] = "Wisps are spawned instead of flies#Up to 5 flies makes 1 wisp with increased health and damage", -- Jar of Flies
-	[437] = "D7 effect when hit for the first time", -- D7
-	[439] = "No special effect", -- Mom's Box
-	[441] = "Lil Brimstone tears (requires releasing the fire button)#All Mega Blast wisps fire continuously during a Mega Blast", -- Mega Blast
-	[475] = "8 high damage wisps", -- Plan C
-	[476] = "Spawns a second wisp if no pickup was duplicated", -- D1
+	[406] = "Middle ring wisp#Random damage and fire rate", -- D8
+	[419] = "Middle ring wisp#20% chance for tears to teleport enemies#Wisps flicker every 5 seconds", -- Teleport 2.0
+	[421] = "Middle ring wisp#{{Charm}} Charm fart when destroyed", -- Kidney Bean
+	[422] = "Middle ring wisp#30% chance for petrifying tears#Respawns all wisps that got destroyed in the previous room", -- Glowing Hourglass
+	[427] = "Bounces around the room, exploding when shot by Isaac#Cannot shoot tears or deal contact damage", -- Mine Crafter
+	[434] = "Middle ring wisp#Wisps are spawned instead of flies#Up to 5 flies makes 1 wisp with increased health and damage", -- Jar of Flies
+	[437] = "Middle ring wisp#D7 effect when hit for the first time", -- D7
+	[439] = "Middle ring wisp#No special effect", -- Mom's Box
+	[441] = "Middle ring wisp#{{Chargeable}} Chargeable Brimstone tears#Shoot continuously during a Mega Blast", -- Mega Blast
+	[475] = "Middle ring wisp#Spawns 8 high damage wisps that stay after death", -- Plan C
+	[476] = "Middle ring wisp#Spawns a second wisp if no pickup was duplicated", -- D1
 	[477] = "Spawns the wisps of all absorbed active items#Doesn't spawn a wisp itself", -- Void
-	[478] = "Pauses all enemies and shots for 3 seconds when destroyed", -- Pause
-	[479] = "Chance to spawn a trinket when destroyed", -- Smelter
-	[480] = "Wisp's health increases the more pickups were destroyed", -- Compost
-	[481] = "5% chance for tears to transform enemies into random wisps", -- Dataminer
+	[478] = "Middle ring wisp#Pauses all enemies and shots for 3 seconds when destroyed", -- Pause
+	[479] = "Middle ring wisp#Chance to spawn a {{Trinket}} trinket when destroyed", -- Smelter
+	[480] = "Middle ring wisp#Only spawns a wisp if at least one pickup got converted#Increased HP for each pickup converted", -- Compost
```

Comment:
change order of informations, to first show the HP increase, and then the limitations